server/server.js

```diff
@@ -177,6 +177,25 @@ app.post("/block-shot", wrap(async (req, res) => {
   res.send(shotId);
 }));
 
+// Monitoring endpoint. Note version.json is generated by CI.
+app.use("/__version__", express.static(path.join(__dirname, "version.json")));
+
+// Monitoring endpoint. Verifies this app is up and accessible.
+app.get("/__lbheartbeat__", function(req, res) {
+  res.sendStatus(200);
+});
+
+// Monitoring endpoint. Verifies this app is up and can connect to the database.
+app.get("/__heartbeat__", async function(req, res) {
+  // TODO: worth it to return the DB patch level?
```

Comment:
I don't think it benefits us much, unless you have a use for it. All we really need is for it to return 200.